### PR TITLE
CART-89 init: Randomize proto query

### PR DESCRIPTION
**Pull request by frostedcmos**
Experiment to randomize protocol query target

Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

File: src/mgmt/cli_mgmt.c [hunk 1]
```diff
@@ -335,10 +342,13 @@ int dc_mgmt_net_cfg(const char *name)
 	Mgmt__GetAttachInfoResp *resp;
 
 	/* Query the agent for the CaRT network configuration parameters */
-	rc = get_attach_info(name, false /* all_ranks */, &info, &resp);
+	rc = get_attach_info(name, true/* all_ranks */, &info, &resp);
```
Comment (mchaarawi) [inline src/mgmt/cli_mgmt.c:345]: if we have 100 clients processes on 1 node, will this generate 100 drpc calls from the agent on that node to the server?

Comment (frostedcmos) [reply src/mgmt/cli_mgmt.c:345]: Per my understanding it should not as agent is supposed to only let 1 through and cache the rest. @kjacque can you verify this for us?

Comment (kjacque) [reply src/mgmt/cli_mgmt.c:345]: The agent caches the first response, yes.

**Review by kjacque — DISMISSED** [review]

**Comment by phender** [comment]
src/include/daos/mgmt.h is missing an update to the copyright.

**Review by mchaarawi — APPROVED** [review]

**Pull request merged**